### PR TITLE
UPBGE: New lod system.

### PR DESCRIPTION
**Pull request by panzergame**

### CODE REVIEW

File: source/gameengine/Converter/KX_BlenderSceneConverter.cpp [hunk 1]
```diff
@@ -142,6 +142,11 @@ KX_BlenderSceneConverter::~KX_BlenderSceneConverter()
 		delete (adtList);
 	}
 
+	for (unsigned int i = 0; i < m_map_blenderobject_to_lodlevels.size(); i++) {
```
Comment (DCubix) [inline source/gameengine/Converter/KX_BlenderSceneConverter.cpp]: can we create a typedef for unsigned types somewhere so it's shorter to type? Like this:
```c
typedef unsigned int uint;
typedef unsigned short ushort;
etc...
```

Comment (panzergame) [inline source/gameengine/Converter/KX_BlenderSceneConverter.cpp]: I will avoid use uint or ushort because we can easily mixed up with short.

**Comment by panzergame** [comment]
I think i'm wrong with names: KX_LodLevel is a bit redundant since Lod is Level of detail. I propose to rename everything:
KX_LodLevels --> KX_Lod
KX_LodLevel --> KX_Lod::Level

Here we can mixed up about the plurial. Also it's is needed to specify that is a mesh/object lod ?
@DCubix : What do you think ?

**Comment by DCubix** [comment]
Yeah, it's just a mesh lod level

**Comment by panzergame** [comment]
pushed in c0e2193b0ed2ed2ffa11f0f737cfa1c1ef87574a

**Pull request closed without merging**